functests/sctp/sctp.go

```diff
@@ -153,33 +154,84 @@ var _ = Describe("sctp", func() {
 				// OCP-26760
 				It("Should connect a client pod to a server pod", func() {
 					testClientServerConnection(client.Client, runningPod.Status.PodIP,
-						activeService.Spec.Ports[0].Port, clientNode, serverPod.Name)
+						30101, clientNode, serverPod.Name, testNamespace)
 				})
 				// OCP-26763
 				It("Should connect a client pod to a server pod. Feature LatencySensitive Active", func() {
 					fg, err := client.Client.FeatureGates().Get("cluster", metav1.GetOptions{})
 					Expect(err).ToNot(HaveOccurred())
 					if fg.Spec.FeatureSet == "LatencySensitive" {
 						testClientServerConnection(client.Client, runningPod.Status.PodIP,
-							activeService.Spec.Ports[0].Port, clientNode, serverPod.Name)
+							30101, clientNode, serverPod.Name, testNamespace)
 					} else {
 						Skip("Feature LatencySensitive is not ACTIVE")
 					}
 				})
 				// OCP-26761
 				It("Should connect a client pod to a server pod via Service ClusterIP", func() {
 					testClientServerConnection(client.Client, activeService.Spec.ClusterIP,
-						activeService.Spec.Ports[0].Port, clientNode, serverPod.Name)
+						activeService.Spec.Ports[0].Port, clientNode, serverPod.Name, testNamespace)
 				})
 				// OCP-26762
 				It("Should connect a client pod to a server pod via Service NodeIP", func() {
 					testClientServerConnection(client.Client, nodes.Items[0].Status.Addresses[0].Address,
-						activeService.Spec.Ports[0].Port, clientNode, serverPod.Name)
+						activeService.Spec.Ports[0].Port, clientNode, serverPod.Name, testNamespace)
 				})
 			})
 			AfterEach(func() {
-				deleteService(client.Client)
-				namespaces.Clean("sctptest", client.Client)
+				deleteService(client.Client, testNamespace)
+				namespaces.Clean(testNamespace, client.Client)
+			})
+		})
+		var _ = Describe("Test Connectivity Default NS", func() {
```

Comment:
Question can we use a ginko table with two strings one for the testname and one for the defaultname?

This looks a bit duplicate

Reply:
Ah, makes sense. Not sure how to handle the polarion annotations in that regard though.

Reply:
@nikita133 any ideas? ^^^^^^^^^^^

Reply:
@fedepaol @SchSeba  We decided to add comment with polarion ID on top of the test. This is temporary solution but this is all what we have at the moment. My manager is trying to handle this issue.

Reply:
So, I tried to rework to table tests, but it's not that easy as the `BeforeEach` depends on the namespace we are working with, and `BeforeEach` can't be run inside a table entry. I'll try again a cleaner approach (maybe refactoring some shared code into functions).